app/Http/Controllers/RecipeController.php

```diff
@@ -29,13 +29,16 @@ class RecipeController extends Controller
      */
     public function index(Request $request)
     {
-        $filteredRecipes = Recipe::forUser()
-            ->Filter($request)
-            ->paginate(10);
+
+        $recipes = Recipe
+            ::ForUser()
+            ->FilterRecipes($request);
```

Comment:
Same as above, filtering should work with params extracted from request so that the method can be reused